### PR TITLE
Fixed or added pictures and planes.

### PR DESCRIPTION
**Pull request by pmd5700**
## Describe your changes
Added Pictures:
96-5302
N30GJ
AE10E7
A340AX

Added Planes:
91-0512
94-00318
00-01051
19-5948
168097

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

File: plane-alert-db.csv [hunk 1]
```diff
@@ -13317,3 +13317,8 @@ AE0C60,93-26485,United States Army,UH-60L Blackhawk,H60,Mil,Get To The Choppa,Mi
 A86908,N641AX,Airborne Tactical Advantage,Dassault Mirage F1 CR,MRF1,Civ,Aggressor Squadron,Callsign Viper,Broke The Hard Deck,Hired Gun,https://atacusa.com/
 AE6CF1,20-1941,USAF,Gulfstream C-37B,GLF5,Mil,Bizjet,Must Be Nice,VIP Transport,USAF,https://www.airforce.com
 A22E3C,N24G,Goodyear Tire & Rubber Company,Gulfstream G150,G150,Civ,Vulcanization,Blimp,Tyres,As Seen on TV,https://w.wiki/4$EU
+ADFD73,91-0512,United States Navy,Swearingen C-26D Metro,SW4,Mil,Rapid Respone,Passenger Transport,Metroliner,United States Navy,https://www.navy.mil
+ADFE4A,94-00318,United States Army,MC-12S Liberty,BE20,Mil,Surveillance,Eye In The Sky,ISR,Oxcart,https://tinyurl.com/2p89t5k4
+AE093A,00-01051,United States Army,Cessna UC-35B Citation,C560,Mil,VIP Transport,Bizjet,Must Be Nice,Toy Soldiers,https://www.army.mil
+AE6D7E,19-5948,USAF,HC-130J Hercules,C30J,Mil,Combat King II,Search And Rescue,Air2Air,USAF,https://en.wikipedia.org/wiki/Lockheed_HC-130
+AE4FB1,168097,United States Navy,Sikorsky MH-60R Seahawk,H60,Mil,CSAR,Maritime Patrol,MEDEVAC,United States Navy,https://en.wikipedia.org/wiki/Sikorsky_SH-60_Seahawk
```
Comment (dziban303) [inline plane-alert-db.csv:13324]:
```suggestion
AE4FB1,168097,United States Navy,Sikorsky MH-60R Seahawk,H60,Mil,ASW,Maritime Patrol,Sub Hunter,United States Navy,https://en.wikipedia.org/wiki/Sikorsky_SH-60_Seahawk#MH-60R
```

Comment (dziban303) [reply plane-alert-db.csv:13324]: If this is a Romeo it's not really doing MEDEVAC or CSAR. I'll change the other MH-60Rs in the db later

File: plane_images.csv [hunk 2]
```diff
@@ -3654,7 +3654,7 @@ A39D53,https://live.staticflickr.com/65535/50627312312_438d2062b4_k.jpg,https://
 A3AC2F,https://cdn.jetphotos.com/full/6/93510_1539217230.jpg,https://www.airport-data.com/images/aircraft/001/434/001434546.jpg,https://www.airfighters.com/photo_9999_276418.jpg,
 A3B39D,https://atacusa.com/images/gallery/hawker-2-slide.jpg,https://live.staticflickr.com/749/23319352000_14fde6ed89_b.jpg,,
 A3B754,https://live.staticflickr.com/65535/51341625084_bc28b3dac7_k.jpg,https://live.staticflickr.com/65535/51341894870_6604c4c0a7_k.jpg,https://live.staticflickr.com/65535/51341624639_f14804a4bc_k.jpg,
-A3BD64,https://live.staticflickr.com/4419/36516547810_e844ef08f6_b.jpg,https://live.staticflickr.com/4432/36516551460_55271df56b_b.jpg,,
+A3BD64,https://live.staticflickr.com/4419/36516547810_e844ef08f6_b.jpg,https://cdn.jetphotos.com/full/2/64459_1242505745.jpg,https://cdn.jetphotos.com/full/4/43192_1391878612.jpg,https://cdn.jetphotos.com/full/5/23866_1655177693.jpg
```
Comment (dziban303) [inline plane_images.csv:3657]: The jetphotos links are wrong. `A3BD64` is a Hawker Hunter. The pics you added are for a USCG Herc, tail 2001, `AE10E7` which we already have three photos for.

File: plane_images.csv [hunk 3]
```diff
@@ -13350,3 +13350,8 @@ AE0C60,,,,
 A86908,https://www.aerialvisuals.ca/Airframe/Gallery/0/117/0000117278.jpg,,,
 AE6CF1,https://cdn.jetphotos.com/full/5/936572_1682227726.jpg,https://cdn.jetphotos.com/full/5/624902_1693395340.jpg,https://cdn.jetphotos.com/full/6/474206_1692422306.jpg,https://cdn.jetphotos.com/full/6/703375_1680243313.jpg
 A22E3C,https://cdn.jetphotos.com/full/5/636741_1679351291.jpg,https://cdn.jetphotos.com/full/5/94752_1597172961.jpg,https://cdn.jetphotos.com/full/3/80083_1394950242.jpg,
+ADFD73,,,,
+ADFE4A,https://cdn.jetphotos.com/full/5/43019_1620254121.jpg,,,
+AE093A,https://cdn.jetphotos.com/full/5/91741_1596229524.jpg,https://cdn.jetphotos.com/full/6/27060_1561296223.jpg,https://cdn.jetphotos.com/full/5/76708_1560450337.jpg,https://cdn.jetphotos.com/full/6/40176_1559225132.jpg
+AE6D7E,https://cdn.jetphotos.com/full/6/977527_1690813947.jpg,https://cdn.jetphotos.com/full/5/841549_1693569145.jpg,https://cdn.jetphotos.com/full/5/487374_1690141414.jpg,https://cdn.jetphotos.com/full/5/608127_1689706531.jpg
```
Comment (dziban303) [inline plane_images.csv:13356]: None of these pics for `AE6D7E` are correct. Each pic is for a different plane and none of them are 19-5948

**Review by dziban303 — CHANGES_REQUESTED** [review]

File: plane_images.csv (hunk 2 above)
Comment (pmd5700) [reply plane_images.csv:3657]: Bad copy and paste on my part. I meant to fix N340AX AND 2001. Should be fixed now.

File: plane_images.csv (hunk 3 above)
Comment (pmd5700) [reply plane_images.csv:13356]: No idea what I was doing here. Should be fixed now. Sorry!

**Review by dziban303 — APPROVED** [review]

**Pull request merged**